MantidPlot/make_package.rb.in

```diff
@@ -213,6 +242,14 @@ if( "@MAKE_VATES@" == "ON" )
     Qt_Executables << " -executable=#{filename}"
    end
 end
+
+qt_folder = findQtFolder()
+if not Dir.exist?("#{qt_folder}/plugins")
+    p "Cannot find the Qt4 plugins folder"
+    p "You probably need to simlink the plugins folder in #{qt_folder}/lib/qt/plugins to #{qt_folder}/plugins"
```

Comment:
Is there an upstream issue with the `qt@4` formula?

Reply:
I've opened an issue with them to report the problem [here](https://github.com/cartr/homebrew-qt4/issues/38). I've also updated the wiki and added a link to that issue there as well.